src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig

```diff
@@ -69,4 +69,9 @@
       </div>
     </form>
   {% endif %}
+  {% if orderForViewing.hasInvoice() %}
+    <div class="form-inline d-inline-block form-horizontal ml-2">
+      <a href="#" class="btn btn-action partial-refund-display">{{ 'Partial refund'|trans({}, 'Admin.Orderscustomers.Feature') }}</a>
```

Comment:
Use a `<button type="button">` instead, then you can drop the useless `href="#"` and the `event.preventDefault();` in the javascript